src/main/native/cpp/jni/CameraServerJNI.cpp

```diff
@@ -1,17 +1,16 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "edu_wpi_cscore_CameraServerJNI.h"
```

Comment:
We should exclude the JNI file until we have a better solution for formatting it.

Reply:
To be clear, I excluded the file again after fixing the formatting.